packages/cluster/src/commands/logs.js

```diff
@@ -4,30 +4,33 @@ const { reporter, exec, tryCatchAsync } = require('@dhis2/cli-helpers-engine')
 const {
     initDockerComposeCache,
     makeComposeProject,
+    makeEnvironment,
     resolveConfiguration,
 } = require('../common')
-const defaults = require('../defaults')
 
-const run = async function({ service, name, cluster, ...argv }) {
-    const {
-        dockerComposeRepository,
-        dockerComposeDirectory,
-    } = resolveConfiguration(argv, {}, cluster)
+const run = async function(argv) {
+    const { service, name } = argv
+    const cfg = await resolveConfiguration(argv)
+
     const cacheLocation = await initDockerComposeCache({
+        composeProjectName: name,
```

Comment:
Should this also be `makeComposeProject(name)` or the parameter just called `name`?  Otherwise maybe `initDockerComposeCache::composeProjectName` is a bit misleading...

Reply:
Yeah, I've considered that too.

The main reason I want `name` here is to make the cache a lot easier to browse: `d2 debug cache list clusters/dev/docker-compose`

If we use `makeComposeProject`: `d2 debug cache list clusters/d2-cluster-dev/docker-compose`

Since all the compose projects are already namespaced to `clusters/{name}` I don't think it makes sense to prefix it again with `d2-cluster-{name}`.

It's still the cache directory for the compose project, so the function name names sense (`initDockerComposeCache`), it's just the argument that is slightly misleading. I can rename it to `composeCacheName` if that makes it clearer.

Reply:
@amcgee Alternatively `initDockerComposeCache::cacheDirName`?

Reply:
We can rename this later but I think `name` might even be fine